command/start.go

```diff
@@ -184,6 +184,19 @@ func (c *startCommand) Run(args []string) int {
 		return ExitCodeParseFlagsError
 	}
 
+	// TODO: Remove this after default subcommands are not supported.
+	// We have to check the length of the config files to ensure we don't print any message
+	// for the plain `consul-terraform-sync` command execution.
+	if len(*c.configFiles) != 0 && *c.isDeprecatedStartUp {
+		c.UI.Warn("====================================================")
+		c.UI.Warn("Warning: Usage of consul-terraform-sync without a subcommand is deprecated and will be removed in a future release. ")
+		c.UI.Warn("")
+		c.UI.Warn("Use `consul-terraform-sync start` instead. Or see `consul-terraform-sync start -help`.")
+		c.UI.Warn("")
+		c.UI.Warn("For additional help, view our documentation: https://www.consul.io/docs/nia/cli/start")
```

Comment:
```suggestion
		c.UI.Warn("Use `consul-terraform-sync start` instead.")
		c.UI.Warn("")
		c.UI.Warn("For additional information, use `consul-terraform-sync start -help` or view the documentation: https://www.consul.io/docs/nia/cli/start")
```

I was thinking that it was more clear to combine the two help lines. Let me know what you think!


```
$ consul-terraform-sync --config-file=config.hcl
====================================================
Warning: Usage of consul-terraform-sync without a subcommand is deprecated and will be removed in a future release.

Use `consul-terraform-sync start` instead.

For additional information, use `consul-terraform-sync start -help` or view the documentation: https://www.consul.io/docs/nia/cli/start
====================================================
```